src/felles-komponenter/EkspanderbartTekstomrade.tsx

```diff
@@ -0,0 +1,45 @@
+import { NedChevron, OppChevron } from 'nav-frontend-chevron';
+import React, { useState } from 'react';
+
+import styles from './EkspanderbartTekstomrade.module.less';
+
+const MoreOrLess = ({ text, maxCharacters }: { text: string; maxCharacters: number }) => {
```

Comment:
Kan disse her trekkes ut i et interface og som const? Også kan man jo vurdere å ha maxCharacters som optional med en default (defualten i eksempelet er bare å vise hvordan det kan gjøres). Dvs. at det blir sånn her:
```suggestion
interface MoreOrLessProps {
  text: string;
  maxCharacters?: number;
}

const MoreOrLess = (props:  MoreOrLessProps) => {
  const { text, maxCharacters = 200 } = props;
```

Reply:
> Kan disse her trekkes ut i et interface og som const? Også kan man jo vurdere å ha maxCharacters som optional med en default (defualten i eksempelet er bare å vise hvordan det kan gjøres). Dvs. at det blir sånn her:
> ```suggestion
> interface MoreOrLessProps {
>   text: string;
>   maxCharacters?: number;
> }
> 
> const MoreOrLess = (props:  MoreOrLessProps) => {
>   const { text, maxCharacters = 200 } = props;
> ```

Hadde det sånn og det er sikkert best practice, men jeg når jeg så på koden så slo det meg at det ble mye interface og boiler platefor noe som kun er tenkt å brukes internt her. Reverterer.